API Blueprint Specification.md

```diff
@@ -478,6 +490,232 @@ Name and description of the API
 
 ---
 
+<a name="def-authentication-section"></a>
+## Authentication section
+- **Parent sections:** none
+- **Nested sections:** [`1` Authentication schemes section](#def-authenticationschemes-section), [`1` Response section](#def-response-section)
+- **Markdown entity:** header
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication` keyword in Markdown header entity.
+
+    # Authentication
```

Comment:
There should never be a case where this is level 1 heading and instead should be 2 (`##`) since it should sit within the API ([API Name section](https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#api-name--overview-section)).